### ISSUE TITLE
Additional information for orders

### ISSUE THREAD

**Issue by alvyn279** [Lercerss/graphelier#110]
As user, I want to be able to view additional details regarding the orders on specific price levels. 

Additional information will include:
- Quantity
- Price
- Last modified date
- Created
- List of messages related to the order

Tasks: #126, #141, #188

| Metric       | Value           |
|--------------|-----------------|
| Story Points | 9 |
| Priority     | Medium |
| Risk         | Medium |

# Tasks
- Extract information from messages and store it in new collections
- Create API route for front-end querying based on order id
- On front-end, request for order information
- Display a new pane that pops up to show this information

# Demo steps
- Generate orderbook
- Click on an order
- View its details

## Customer signoff
- [ ] Pierre-Marc Fournier
- [ ] TA

**Comment by chimano** [comment]
Here's the API contract 
```
URL: /order/{instrument}/{order_id}/{timestamp}

Example request: /order/SPY/15724/1233213131131

Response object:
{
    "instrument": string,
    "id": int,
    "quantity": int,
    "price": int,
    "last_modified": string,
    "created_on": string,
    "is_executed": string,
    "messages": [
       {"instrument": string,
        "timestamp": string,
        "message_type": int,
        "order_id": int,
        "share_qty": int,
        "price": float,
        "direction": int,
        "sod_offset": string
      },...]
}
NOTE: messages will always be sorted in ascending order using timestamp
```

**Comment by chimano** [comment]
Ah right I forgot it can have multiple executes. Do you want me to remove the field?

**Comment by riyaGit** [comment]
### Acceptance test
| Steps                    | Inputs | Expected Outputs                                          | Status |
|--------------------------|--------|----------------------------------------------------------|--------|
| Go to main page  | N/A    | N/A | N/A   |
|  Select a datetime to generate an orderbook  |  Click on an order | A drawer should popup containing all the order information and the order that is clicked on should change colour | PASS |
|  Click anywhere on the screen outside the drawer |  N/A | The drawer popup should disappear | PASS |

**Comment by chimano** [comment]
Can we close this?

**Comment by alvyn279** [comment]
> Can we close this?

think we were leaving it open for the click to message task

**Comment by riyaGit** [comment]
Addressed in pull requests #205 #187 #154
